libcxx/include/__format/format_arg_store.h

```diff
@@ -228,15 +228,15 @@ _LIBCPP_HIDE_FROM_ABI void __store_basic_format_arg(basic_format_arg<_Context>*
   ([&] { *__data++ = __format::__create_format_arg<_Context>(__args); }(), ...);
 }
 
-template <class _Context, size_t N>
+template <class _Context, size_t _Nargs>
```

Comment:
This is our typical name for these.
```suggestion
template <class _Context, size_t _Np>
```